### PR TITLE
Migrate to Travis

### LINKED ISSUE #58: Migrate to Travis CI
**Is your feature request related to a problem? Please describe.**
In order to ease the development workflow. It would be more convenient if we moved to Travis CI.
I will be working on this issue

### PR DESCRIPTION
**Pull request by RaghavAwasthi**
This PR will fix #58 
Travis Configuration File has been Added.
The code works absolutely fine. You can have a look at my Fork for live demo.
## Additional Step
@krharsh17 
Please activate Travis from your Admin Account for Builds to Auto Trigger

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+    - $HOME/.android/build-cache
+
```
Comment (ShivangiSingh17) [inline .travis.yml]: @RaghavAwasthi Please parse yml file through [YAML Formatter](https://jsonformatter.org/yaml-formatter), the .travis.yml have basic indentation related syntax issues.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
```
Comment (ShivangiSingh17) [inline .travis.yml]: Remove this extra space.

**Review by ShivangiSingh17 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (RaghavAwasthi) [reply .travis.yml]: Done

File: .travis.yml (hunk 1 above)
Comment (RaghavAwasthi) [reply .travis.yml]: Are you sure that there are syntax issues?
As I tested it on my Fork with Travis and it is successfully building as per requirements.
Build link:-https://travis-ci.com/github/RaghavAwasthi/barview-android/builds/154100914
Anyways I have parsed the yml file as you requested.

**Review by ShivangiSingh17 — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (ShivangiSingh17) [reply .travis.yml]: Yes I was sure thats why I suggested you. In long run of the project the travis should be proper.

**Pull request merged**